my-code/whats-in/src/pages/Homepage/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import iconSearch from '../../images/icon-magnifier-grey.svg'
+import iconLike from '../../images/icon-heart-grey.svg'
+import iconLiked from '../../images/icon-heart-full.svg'
+import Loading from '../../components/Loading'
+import Empty from '../../components/Empty'
+import NotFound from '../../components/Notfound'
+
+export default function Homepage({ handleSearch, movies, handleMovieSelection, isLoading, handleLike, isMovieLiked, isEmptySearch }) {
```

Comment:
because of the architecture you chose of having all state in your App component, you're receiving a lot of props here when, in fact, this component could be independent with self-contained state.

Even if you still chose to go with option A with no routing, by inferring some logic based on state you already have you could simplify a bit the amount of props received.